hhnk_research_tools/waterschadeschatter/wss_main.py

```diff
@@ -75,11 +74,11 @@ def run(self,
         Calculation type options: 'sum','direct','indirect'
         """
 
-        if output_raster.exists:
+        if output_raster.exists():
             if overwrite is False:
                 return
             else:
-                output_raster.pl.unlink()
+                output_raster.unlink()
 
         #Create output raster
         output_raster.create(metadata=self.depth_raster.metadata,
```

Comment:
@ L171: `self` is geen lekkere lokale variabele
`
self = Waterschadeschatter(depth_file=depth_file, 
                                landuse_file=landuse_file, 
                                wss_settings=wss_settings,
                                )
`

netter = `self` vervangen voor `wss` en  `wss.run(....` i.p.v. `self.run(....` gebruiken op L177.

Reply:
voor debuggen en ontwikkeling is het wel heel fijn omdat je dan de code binnen de class gewoon kan draaien.